pages/index.js

```diff
@@ -69,7 +68,7 @@ export default function Index() {
               </div>
             </div>
             <p>
-                We adopt the SSB <q>social overlay</q> paradigm to establish a peer-to-peer replication layer on top of distributed version control systems. We model this largely based on an (unmodified) git backend, but strive for the protocol to be general enough to allow implementations on top of different paradigms, e.g pijul. We assume a mostly non-adversarial setting. Features requiring strong sybil resistance (or are otherwise difficult to implement in a purely peer-to-peer way) are out of the scope of this specification.
+                Code is shared and collaborated on in a peer-to-peer replication layer built on top of a distributed version control system. We assume a mostly non-adversarial setting. Features requiring strong sybil resistance (or are otherwise difficult to implement in a purely peer-to-peer way) are out of the scope of this specification.
```

Comment:
Neither the original nor this modification really explains how it works. I can give it a shot.

Reply:
Yeah, that would be great. I was going to run through it with Kim, that's why I stopped making mods. If you take a shot we can go from there.